pages/common/hledger-ui.md

```diff
@@ -0,0 +1,32 @@
+# hledger-ui
+
+> A robust, friendly plain text accounting app (the terminal UI).
```

Comment:
Hi @kbdharun , thank you for the edits!

Re this one, I tried to make the wording consistent across the three, since they are really different UIs of the same app. Do you still prefer yours here ?

Reply:
> Re this one, I tried to make the wording consistent across the three, since they are really different UIs of the same app. Do you still prefer yours here ?

Hi, I have updated my suggestion (after going through the other descriptions again), I have separated the distinction line and the generic description [as we can have 2 descriptions].

Also in the `hledger` page I removed the CLI mention as we don't mention it specifically in our pages [as by default it is implied that it's a CLI app, but we do mention if it is TUI or GUI. Additionally, I added a see also reference to the new pages.